src/tactic/core.lean

```diff
@@ -63,6 +63,12 @@ meta def mfoldl {α : Type} {m} [monad m] (f : α → expr → m α) : α → ex
 | x e := prod.snd <$> (state_t.run (e.traverse $ λ e',
     (get >>= monad_lift ∘ flip f e' >>= put) $> e') x : m _)
 
+/-- Strip-away the context-dependent unique id for the given local const and return: its friendly
+`name`, its `binder_info`, and its `type : expr`.-/
+meta def local_const_get_kind : expr → name × binder_info × expr
```

Comment:
This should live in `meta/expr.lean`, and maybe be named `get_local_const_kind`? (I don't really care but the name reads funny to me.)

Reply:
Sure! forgot about that file